src/cc_catalog_airflow/dags/provider_api_scripts/jamendo.py

```diff
@@ -0,0 +1,322 @@
+"""
+Content Provider:       Jamendo
+
+ETL Process:            Use the API to identify all CC-licensed images.
+
+Output:                 TSV file containing the image, the respective
+                        meta-data.
+
+Notes:                  https://api.jamendo.com/v3.0/tracks/
+                        No rate limit specified.
+"""
+import os
+import logging
+from urllib.parse import urlparse
+
+from common import DelayedRequester, AudioStore
+from common.licenses.licenses import get_license_info
+from common.urls import rewrite_redirected_url
+from util.loader import provider_details as prov
+
+# On Jamendo Music, you can enjoy a wide catalog of more than 500,000 tracks
+# shared by 40,000 artists from over 150 countries all over the world.
+
+# Audio quality: uploaded as WAV/ FLAC/ AIFF
+# bit depth: 16/24
+# sample rate: 44.1 or 48 kHz
+# channels: 1/2
+
+logging.basicConfig(
+    format='%(asctime)s - %(name)s - %(levelname)s:  %(message)s',
+    level=logging.INFO
+)
+logger = logging.getLogger(__name__)
+
+LIMIT = 200  # number of items per page in API response
+DELAY = 1  # in seconds
+RETRIES = 3
+
+HOST = 'jamendo.com'
+ENDPOINT = f'https://api.{HOST}/v3.0/tracks'
+PROVIDER = prov.JAMENDO_DEFAULT_PROVIDER
+APP_KEY = os.getenv('JAMENDO_APP_KEY', 'dd3c1077')
```

Comment:
Leaving a default key made it possible to run the script quickly to test but we should remove it, this will avoid forgetting to use a secret key in production later.

Reply:
🙈